inferno-lsp/inferno-lsp.cabal

```diff
@@ -70,4 +71,6 @@ executable inferno-lsp-server
   ghc-options: -Wall -Wincomplete-uni-patterns -Wincomplete-record-updates -threaded -rtsopts -with-rtsopts=-T
   build-depends:
       base >=4.7 && <5
+    , inferno-core
```

Comment:
Btw, we need version bounds on all dependencies in order to upload to hackage. (We can add this before the next update)